### DISCUSSION TITLE
RCKA keys

### DISCUSSION THREAD

**Discussion by richard-sisson** [australiangeomechanics/agsformat#55] [Q&A] [answered]
Hi,

We noted that RCKA (alteration) specifies LOCA_ID, RCKA_TOP and RCKA_BASE as key headings.

This differs from RCKS, SOLC, SOLM that require LOCA_ID and ?_TOP.

Was this intentional?

Thanks
Richard

**Comment by SolidRockProgrammer** [comment, marked as answer]
I think this was unintentional and not needed. Resulted from copying WETH

**Reply by richard-sisson** [reply]
Thanks Phil
